Tests/ReactiveSwiftTests/SignalProducerSpec.swift

```diff
@@ -2696,6 +2721,6 @@ extension SignalProducer {
 			}
 		}
 
-		return SignalProducer.attempt(operation)
+		return SignalProducer(operation)
```

Comment:
I prefer the `static func attempt` name for that reason. It also lends itself to RACformatting.

```swift
SignalProducer
    .attempt {
        …
    }
    .flatMap(.concat) {
        …
    }
```

Is renaming this an essential part of the PR? If not, could we move it off the protocol here and discuss the renaming in a separate PR?